documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -233,12 +238,26 @@ env:
 
 === Control Plane listener feature gate
 
-When enabled, the `ControlPlaneListener` feature gate configures Kafka brokers to use a separate listener for controller connections and for data connections (replication).
-This feature gate is currently in the alpha phase and is disabled by default.
-To enable it, use `+ControlPlaneListener` in the `STRIMZI_FEATURE_GATES` environment variable in the Cluster Operator configuration.
-This feature gate has to be disabled when upgrading from Strimzi 0.22 and earlier or when downgrading to Strimzi 0.22 and earlier.
+You can use the `ControlPlaneListener` feature gate to change the communication paths used for inter-broker communications.
+
+When `ControlPlaneListener` is enabled, Kafka brokers use a dedicated internal _control plane listener_ for controller connections (that is, traffic from the Kubernetes control plane) between brokers. 
+The control plane listener uses port 9090. 
+Data connections (data replication between the Kafka brokers) continues to use the internal listener on port 9091. 
+
+When `ControlPlaneListener` is disabled, controller and data connections go through the same internal listener on port 9091. 
+This was the default behavior before the feature gate was introduced.
+
+Control plane listeners might improve performance because important requests from the control plane, such as partition leadership changes, are not delayed by data plane traffic, such as data replication.
```

Comment:
```suggestion
Control plane listeners might improve performance because important requests from the control plane, such as partition leadership changes, are not delayed by data plane traffic, such as data replication across clusters.
```

Reply:
If I understand correctly, the data replication is across Kafka brokers.